pkg/forklift-api/webhooks/mutating-webhook/mutators/ovirt-cert-mutator_test.go

```diff
@@ -0,0 +1,36 @@
+package mutators
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/onsi/gomega"
+)
+
+func TestCertAppending(t *testing.T) {
+	g := gomega.NewGomegaWithT(t)
+
+	providedCa, err := ioutil.ReadFile("completeCerts.pem")
+	if err != nil {
+		g.Fail("Failed to read certificate file")
+	}
+
+	newCa, err := ioutil.ReadFile("engineCert.pem")
+	if err != nil {
+		g.Fail("Failed to read engine certificate file")
+	}
+
+	//Test the case where two certificates are identical but have a different line count due to new lines.
+	g.Expect(contains(providedCa, newCa)).To(gomega.Equal(true))
```

Comment:
one of the nice things about gomega is that you can write things in a readable way and having that `gomega.` everywhere makes it less readable. how about changing the import to `. "github.com/onsi/gomega"` and then change this to `g.Expect(contains(providedCa, newCa)).To(BeTrue()))` (and drop 'gomega.' below)

Reply:
nice!